piksi_tools/heartbeat.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Colin Beighley <colin@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from sbp.system import SBP_MSG_HEARTBEAT, MsgHeartbeat
+
+class Heartbeat(object):
```

Comment:
I still think the link and callback stuff should be left out of the Heartbeat thing (see previous comment, which has since disappeared). We have an existing pattern of simple callbacks managed by at a high-level by the app (https://github.com/swift-nav/piksi_tools/blob/4d4dc570433ecaf190b6c4631121cccb2ef33d46/piksi_tools/serial_link.py#L171) that should probably be followed here. If it's not doing any resource management, it probably doesn't need context management either (which should simplify the code using it below).